spec/controllers/approval_requests_controller_spec.rb

```diff
@@ -0,0 +1,346 @@
+require 'rails_helper'
+
+RSpec.describe ApprovalRequestsController do
+  let(:team) { create :team, :with_users, number_of_members: 1 }
+  let(:manager) { team.team_roles.managers.first.user }
+  let(:member)  { team.team_roles.members.first.user }
+  let(:guest)   { team.team_roles.guests.first.user }
+  let(:user)    { manager }
+  let(:approval_request) { manager.approval_requests.first }
+  let(:create_vacation_request) do
+    create :vacation_request, :with_approval_requests, user: member
+  end
+
+  before { create_vacation_request }
+
+  RSpec.shared_examples 'request with conflict' do
+    it 'should respond with status code :conflict (409)' do
+      send_request
+      expect(response).to have_http_status(:conflict)
+    end
+
+    it 'should not remove approval request from DB' do
+      expect { send_request }.not_to change(ApprovalRequest, :count)
+    end
+
+    it 'should not change vacation request status' do
+      id = approval_request.vacation_request_id
+
+      expect { send_request }
+        .not_to change { VacationRequest.find_by(id: id).status }
+    end
+  end
+
+  ################################################################## GET #accept
+  describe 'GET #accept' do
```

Comment:
If you rally do want to use `RSpec.shared_examples` and 'RSpec.describe', please use it in all places.